### ISSUE TITLE
confusing log messages

### ISSUE THREAD

**Issue by elias-putz-kiwigrid** [lukas-krecan/ShedLock#138]
```
16:12:32 DEBUG --- DefaultLockingTaskExecutor : Unlocked export_cronjob.
16:12:52 DEBUG --- DefaultLockingTaskExecutor : Not executing export_cronjob. It's locked.
```

![well yes, but actually no](https://i.kym-cdn.com/entries/icons/original/000/028/596/dsmGaKWMeHXe9QuJtq_ys30PNfTGnMsRuHuo_MUzGCg.jpg)

The execution is still locked because `lockAtLeastFor` is set pretty high, so everything is working as intended, just the log messages are a bit confusing.

**Comment by lukas-krecan** [comment]
Thanks for feedback, you are right, would something like this help https://github.com/lukas-krecan/ShedLock/pull/139 ?

**Comment by elias-putz-kiwigrid** [comment]
Yes, that should make it more clear whats happening :)

**Comment by lukas-krecan** [comment]
Released as 2.5.0
